### PR TITLE
BPM Engine decoupling

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

File: src/main/java/org/entando/plugins/pda/core/engine/Engine.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.util.Optional;
+
+@Getter
+@AllArgsConstructor
+public abstract class Engine {
+    protected String type;
+    protected TaskService taskService;
+
+    //protected CommentService commentService;
```
Comment (werbth) [inline src/main/java/org/entando/plugins/pda/core/engine/Engine.java]: Remove commented code

File: src/main/resources/messages.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+org.entando.error.engine.notFound=Engine not supported
+org.entando.error.engine.notSupported=Engine not supported for this operation
+
+org.entando.error.task.notFound=Task not found
```
Comment (werbth) [inline src/main/resources/messages.properties]: Add new line at the end of the file.

File: target/classes/messages.properties [hunk 3]
```diff
@@ -0,0 +1,4 @@
+org.entando.error.engine.notFound=Engine not supported
+org.entando.error.engine.notSupported=Engine not supported for this operation
+
+org.entando.error.task.notFound=Task not found
```
Comment (werbth) [inline target/classes/messages.properties]: Add new line at the end of the file.

**Review by werbth — CHANGES_REQUESTED** [review]

File: target/classes/messages.properties (hunk 3 above)
Comment (ffleandro) [reply target/classes/messages.properties]: actually this shouldn't be commited, it's build files

**Pull request closed without merging**